helm/charts/stan/values.yaml

```
@@ -43,6 +43,22 @@ stan:
   #    name: nats-sys-creds
   #    key: sys.creds
 
+  auth:
+    enabled: false
+
+    # Username/password is used to connect to a NATS Server when authentication with multiple users is enabled
+    # username:
+    # password:
+
+    # Authentication token if the NATS Server requires a token
+    # token:
+
+    # Secret to an NKey seed file if NKey authentication is used
+    # nkey_seed_file:
```

Ok to make these camel case?
```suggestion
    # nkeySeedFile:
```

---

Absolutely! Slipped through.